server/store/missions.js

```diff
@@ -51,13 +51,13 @@ const createMission = async ({ user_id, bid_id }) => {
     'price', price,
     'time_to_pickup', time_to_pickup,
     'time_to_dropoff', time_to_dropoff,
-    'request_id', request_id,
-    'pickup_lat', pickup_lat,
-    'pickup_long', pickup_long,
-    'dropoff_lat', dropoff_lat,
-    'dropoff_long', dropoff_long,
-    'requested_pickup_time', requested_pickup_time,
-    'size', size,
+    'need_id', need_id,
```

Comment:
We are updating the communication docs from `request_uid` to `need_id` as you suggest here 👍

Reply:
Great!